package.json

```diff
@@ -1,38 +1,44 @@
 {
-  "name": "bookstore",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.17.0",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.4"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+	"name": "bookstore",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@testing-library/jest-dom": "^5.17.0",
+		"@testing-library/react": "^13.4.0",
+		"@testing-library/user-event": "^13.5.0",
+		"prop-types": "^15.8.1",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
+		"react-redux": "^8.1.2",
+		"react-scripts": "5.0.1",
+		"uuid": "^9.0.0",
+		"web-vitals": "^2.1.4"
```

Comment:
- You have done great so far. :+1: Your redux store is set up correctly but there are a few requirements you need to meet before moving to the next milestone. :+1: 

> Add Redux Toolkit (npm install react-redux @reduxjs/toolkit).

- [ ] As stated in the project requirement, you need to install `react-redux and @reduxjs/toolkit`. currently, you are importing `createSlice and configureStore` from `@reduxjs/toolkit` but @reduxjs/toolkit is not in your dependencies. I'll suggest you install the package as required to avoid errors due to missing node packages. :+1: :smile: :+1: